packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -97,4 +146,15 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
       updatedAt: now
     })
     .run()
+
+  const billingLeaders = await getBillingLeaders(orgId, dataLoader)
+  const billingLeadersIds = billingLeaders.map((billingLeader) => billingLeader.id)
+
+  // Enable usage stats
+  if (organization.activeDomain) {
+    await enableUsageStats(billingLeadersIds, orgId)
+
+    // Send push notification
+    await sendWebsiteNotifications(orgId, billingLeadersIds, dataLoader)
```

Comment:
Does the `sendWebsiteNotifications` need to wait for `enableUsageStats` to complete? If not, we can use `Promise.all` to speed things up

Reply:
I think it needs to wait, as it sends a notification to usage stats link, so it needs to be enabled first